app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Sign up</h2>
+
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
```

Comment:
I think we need to add the username field here, as we have set it as compulsory since we need to display it when a user creates a "call to action".

````
<%= f.input :username, required: true %>
````